src/client/mod.rs

```diff
@@ -77,12 +79,22 @@ impl ClientBuilder {
         self
     }
 
+    /// Setup SASL username and password. Mechanism is assumed to be PLAIN.
+    pub fn sasl_config(mut self, username: &str, password: &str) -> Self {
+        self.sasl_config = Some(SaslConfig::Plain{
+            username: username.to_string(),
+            password: password.to_string(),
+        });
+        self
+    }
```

Comment:
I would take `SaslConfig` as an argument here so we can extend the config in the future w/o overloading the `ClientBuilder` w/ too many methods.

Reply:
Had to `pub use crate::connection::SaslConfig` in `client/mod.rs` to publicly expose `SaslConfig`